xbmc/pvr/epg/Epg.h

```diff
@@ -219,9 +219,10 @@ namespace PVR
     /*!
      * @brief Delete this table from the given database
      * @param database The database.
+     * @param bCommit queue only or queue and commit
      * @return True if the table was deleted, false otherwise.
      */
-    bool Delete(const std::shared_ptr<CPVREpgDatabase>& database);
+    bool Delete(const std::shared_ptr<CPVREpgDatabase>& database, bool bCommit);
```

Comment:
As far as I can see, this method is no longer called like before, so the sync code path can be removed from its implementation. I would also rename the method to QueueDeleteQuery to make clear what it does.